torchvision/datasets/phototour.py

```diff
@@ -54,26 +55,28 @@ class PhotoTour(VisionDataset):
             'fdd9152f138ea5ef2091746689176414'
         ],
     }
-    mean = {'notredame': 0.4854, 'yosemite': 0.4844, 'liberty': 0.4437,
-            'notredame_harris': 0.4854, 'yosemite_harris': 0.4844, 'liberty_harris': 0.4437}
-    std = {'notredame': 0.1864, 'yosemite': 0.1818, 'liberty': 0.2019,
-           'notredame_harris': 0.1864, 'yosemite_harris': 0.1818, 'liberty_harris': 0.2019}
+    means = {'notredame': 0.4854, 'yosemite': 0.4844, 'liberty': 0.4437,
```

Comment:
this is in theory a BC-breaking change.

I think it would be better to avoid renaming this in this PR

Reply:
Is it BC breaking? I mean after the assignment (L75-76) the dicts are no longer accessible. This only matters if one uses the dicts from the class directly `PhotoTour.mean` or `PhotoTour.std`.

Reply:
Oh, I just saw this, yeah, this was most probably an oversight in the original implementation, good catch!